### DISCUSSION TITLE
Any tips for getting started?

### DISCUSSION THREAD

**Discussion by jfrux** [getndazn/kopytko-framework#22] [Q&A]
Trying to find a framework for testing in a new project and am curious what the recommended methods are for getting started with this project?

**Comment by tbazelczuk** [comment]
Hi,

You can start new project with [kopytko-cli](https://github.com/getndazn/kopytko-cli). Use `kopytko new my-project` to create scaffold project.

**Comment by pawelhertman** [comment]
There is also an example app: https://github.com/pawelhertman/kopytko-example-app
